docs/tutorials/create-an-ethereum-processing-squid.md

```diff
@@ -0,0 +1,455 @@
+---
+id: create-an-ethereum-processing-squid
+title: Create an Ethereun-processing Squid
+description: >-
+  Create a simple squid indexing smart contract data on Ethereum Mainnet
+sidebar_position: 3
+---
+
+# Create an EThereum-processing Squid
+
+## Objective
+
+This tutorial will take the Squid Ethereum template and go through all the necessary steps to customize the project, in order to interact with a different Squid Archive, synchronized with the Ethereum blockchain, and process data from a specific contract ([the Exosama NFT collection](https://exosama.com/)), with substantial configuration changes to what is defined in the template.
+
+The business logic to process these contract is basic, and that is on purpose since the Tutorial aims show a simple case, highlighting the changes a developer would typically apply to the template, removing unnecessary complexity.
+
+In this example  we will be connecting to the [Ethereum mainnet](https://ethereum.org/en/developers/docs/networks/#ethereum-mainnet) and the final objective will be to show the tokens that are part of the contract, who owns them and every time they have been transfered.
+
+## Pre-requisites
+
+The minimum requirements to follow this tutorial are the basic knowledge of software development, such as handling a Git repository, a correctly set up [Development Environment](/tutorials/development-environment-set-up), basic command line knowledge and the concepts explained in this documentation.
+
+## Fork the template
+
+The first thing to do, although it might sound trivial to GitHub experts, is to fork the repository into your own GitHub account, by visiting the [repository page](https://github.com/subsquid/squid-ethereum-template) and clicking the "Use this template" button:
+
+![How to fork a repository on GitHub](</img/.gitbook/assets/Screenshot-2022-02-02-111440.png>)
+
+Next, clone the created repository (be careful of changing `<account>` with your own account)
+
+```bash
+git clone git@github.com:<account>/squid-ethereum-template.git
+```
+
+For reference on the complete work, you can find the entire project [here](https://github.com/subsquid/squid-ethereum-example).
+
+## Define Entity Schema
+
+The next thing to do, in order to customize the project for our own purpose, is to make changes to the schema and define the Entities we want to keep track of.
+
+Luckily, the EVM template already contains a schema that defines the exact entities we need for the purpose of this guide. For this reason, changes are necessary, but it's still useful to explain what is going on.
+
+To index ERC-721 token transfers, we will need to track:
+
+* Token transfers
+* Ownership of tokens
+* Contracts and their minted tokens
+
+And the `schema.graphql` file defines them like shis:
+
+```graphql
+type Token @entity {
+  id: ID!
+  owner: Owner
+  uri: String
+  transfers: [Transfer!]! @derivedFrom(field: "token")
```

Comment:
why `[Transfer!]!` here

Reply:
Literally copied over from here https://github.com/subsquid/squid-evm-template/blob/master/schema.graphql#L5
What do you suggest, instead?